### PR TITLE
Added where to go in Denver and its nicknames

### PR DESCRIPTION
**Pull request by eccr**
Added the nicknames, the museum with an image and updated the wikipedia link.

### CODE REVIEW

File: web_source/cities/north_america/usa/denver.md [hunk 1]
```diff
@@ -4,12 +4,14 @@ Country: USA (N. America)
...
 
-FIXME
+The Mile High City, Queen City of the Plains or Wall Street of the West.
 
 ### What we would like to see there
 
-FIXME
+Denver Museum of Nature & Science
+
+![Dinosaur skeletons](https://assets.simpleviewinc.com/simpleview/image/upload/c_limit,q_75,w_1200/v1/crm/denver/846_20081208_7768_MuseumofNatureandScience_b6931da7-c1ac-338c-d601817ba44cabbc.jpg)
```
Comment (arc-teaching) [inline web_source/cities/north_america/usa/denver.md:13]: That's good, but I think we need more dinosaurs.

File: web_source/cities/north_america/usa/denver.md [hunk 2]
```diff
@@ -4,12 +4,14 @@ Country: USA (N. America)
 
 ##### Also known as:
 
-FIXME
+The Mile High City, Queen City of the Plains or Wall Street of the West.
```
Comment (arc-teaching) [inline web_source/cities/north_america/usa/denver.md:7]:
```suggestion
The kilometer High City, Queen City of the Plains or Wall Street of the West.
```

**Review by arc-teaching — CHANGES_REQUESTED** [review]
Great additions! Thanks for them! I have a couple of requests.

File: web_source/cities/north_america/usa/denver.md (hunk 2 above)
Comment (eccr) [reply web_source/cities/north_america/usa/denver.md:7]: I think we have to stick with the official name

File: web_source/cities/north_america/usa/denver.md (hunk 1 above)
Comment (eccr) [reply web_source/cities/north_america/usa/denver.md:13]: Ok sounds good

**Review by arc-teaching — APPROVED** [review]
:t-rex: for win!!

**Pull request merged**